kirtonBCIlab/boccia-bci
# 145-BCI-selection

Implementation
Connected the `SPO.OnSelectedEvent` listener to the segment clicks and the SPO buttons in play and virtual play

Testing
- Start the game and in game options change the fine fan to a 2 x 2 grid
- Go to virtual play and select a fan segment to switch from the coarse fan to the fine fan
- Without stopping the game, use the search function in the hierarchy and search for "SPO"
- Using the scene view, select the button of interest from the hierarchy. You need the scene view to mark in green which button you'll be pressing in the game.
- While the button of interest is selected, scroll down on the inspector and look for the `SPO/Selectable pool index` option.
- Go back to the game and press "S" to start the stimulus
- The `SPO/Selectable pool index` will change, press the new number in your keyboard
- Once the flashing animation is done, the button of interest should be pressed and the corresponding action should ocurr
